content/articles/understanding-the-51-attack-on-blockchain/index.md

```diff
@@ -0,0 +1,136 @@
+
+The developers of BTG blockchain wanted to achieve decentralization by using GPUs for mining instead of [ASIC](https://www.investopedia.com/terms/a/asic.asp) devices. However, an unknown miner managed to control more than 51% of the overall BTG hashrate leading to the attack.
+
+In the year 2020, BTG suffered yet another 51% attack. The blockchain experienced two reorganizations in two days that let it double-spend, totalling a vast amount of money. The BTG community begged the blockchain to change its algorithm to a more secure one. They suspected there might be secret ASIC mining devices on the BTG network.
+
+#### Ethereum Classic
+In 2020, [ETC](https://ethereumclassic.org/) blockchain faced three consecutive 51% attacks in the same month. Ethereum classic uses PoW consensus algorithm same as Bitcoin. A 51% attack is expensive when performed on large networks like Bitcoin since a tremendous amount of computing power is required to do it. ETC hash rate is lower hence vulnerable to 51% attacks.
+
+The decentralized nature of the ETC PoW makes it difficult to avoid or mitigate the 51% attacks. The attacks were reported not to have had any significant impacts on the ETC prices, but the users' trust was reduced.
+
+### Ways to prevent a 51% attack
+#### 50% Limit on a single miner
+The blockchain should ensure that no single miner or group of miners controls more than 50% of the hashing power. It would be impossible for a single miner or a group to attack the network by outbuilding the longest validated blockchain. To achieve the attack, it would mean that the attacker has to own powerful hardware and requires much energy. Also, an attacker may need luck since the mining process would be random.
+
+A good example is Bitcoin, where its network and hash rate are vast and complex enough to be a substantial initial investment for an attacker to rent mining equipment. Ethereum Classic, on the other hand, is more prone to the attack since its overall hash rate is relatively small compared to Bitcoin.
+
+#### Using Proof of Stake
+A single miner can become the majority player in a small blockchain network.  All the blockchain networks that use PoW have a policy that the miners have to upgrade their equipment regularly. Failure to which, they may not receive the block rewards, and they will fall behind other miners in the network.
+
+In order to avoid the risk of a 51% attack, the blockchain can use [Proof of Stake](https://www.fool.com/investing/stock-market/market-sectors/financials/cryptocurrency-stocks/proof-of-stake) (PoS), which is a more secure consensus than PoW. In most cases, the PoS incentives are controlled by most affluent users unlikely to perform the attack. However, blockchains have moved from this structure, and they prefer more decentralized alternatives such as [Delegated-Proof-of-Stake](https://academy.binance.com/en/articles/delegated-proof-of-stake-explained) (DPoS).
```

Comment:
```suggestion
To avoid the risk of a 51% attack, the blockchain can use [Proof of Stake](https://www.fool.com/investing/stock-market/market-sectors/financials/cryptocurrency-stocks/proof-of-stake) (PoS), which is a more secure consensus than PoW. In most cases, the PoS incentives are controlled by most affluent users unlikely to perform the attack. However, blockchains have moved from this structure, and they prefer more decentralized alternatives such as [Delegated-Proof-of-Stake](https://academy.binance.com/en/articles/delegated-proof-of-stake-explained) (DPoS).
```